core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
Ah -- to clarify, there should not be any conceptual difference between isHidden and isShown. So changing from isShown to isHidden should not affect the underlying concept. If it does then I think the concept is muddy and that is what needs fixing.

Now, assuming we have the underlying concept clear (i.e. _what_ exactly is being shown or hidden), do we go with isShown or isHidden? Tackling that question separately, isShown is probably better, because it avoids "double negative" constructions like "!isHidden".